src/zookeeper/zookeeper_session.cpp

```diff
@@ -24,35 +24,55 @@
  * THE SOFTWARE.
  */
 
+#include <sasl/sasl.h>
 #include <stdlib.h>
 #include <zookeeper/zookeeper.h>
 #include <algorithm>
 #include <utility>
 
 #include "runtime/app_model.h"
 #include "runtime/rpc/rpc_address.h"
+#include "utils/filesystem.h"
 #include "utils/flags.h"
 #include "utils/fmt_logging.h"
+#include "utils/strings.h"
 #include "zookeeper/proto.h"
 #include "zookeeper/zookeeper.jute.h"
 #include "zookeeper_session.h"
 
 DSN_DECLARE_bool(enable_zookeeper_kerberos);
-DSN_DEFINE_string(security,
```

Comment:
It's better to keep these configs, `sasl_service_name` and `sasl_service_fqdn` are acceptable to be added as well. You can:
1. Check `zookeeper_kerberos_service_name` and `zookeeper_sasl_service_fqdn` must be empty, or they are equal to `sasl_service_name/fqdn`.
2. Add "deprecated" to the description of the old configs.
3. If the old configs are not empty, log warning logs to notify that they are deprecated.

Reply:
https://github.com/apache/incubator-pegasus/commit/966c5b831d0a6b9742aea60786c46123f4a053bd

There 2 conifgs just add at last month. And there is no issue involved. 
Is it necessary to do this work? This leads to more difficult to read configurations.